pkg/serverless/trace/trace.go

```diff
@@ -63,6 +66,7 @@ func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load) {
 			s.ta = agent.NewAgent(context, tc)
 			s.spanModifier = &spanModifier{}
 			s.ta.ModifySpan = s.spanModifier.ModifySpan
+			s.ta.FilterSpan = filterSpanFromLambdaLibraryOrRuntime
```

Comment:
We may need to refactor this in the future if we want to support multiple filters

Reply:
Agreed, I think when the time is right we can refactor this :+1: